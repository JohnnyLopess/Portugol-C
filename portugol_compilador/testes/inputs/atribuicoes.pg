programa{
    funcao inicio(){
        inteiro a, b
        a = 1
        b = 2
        inteiro subtracao = b - a
        escreva(subtracao)
        inteiro x = 10, y = 20
        inteiro soma = x + y
        escreva(soma)
    }
}