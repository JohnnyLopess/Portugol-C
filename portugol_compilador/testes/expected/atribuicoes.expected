#include <stdio.h>
#include <string.h>

int main() {
    int a, b;
    a = 1;
    b = 2;
    int subtracao = b - a;
    printf("%d\n", subtracao);
    int x = 10, y = 20;
    int soma = x + y;
    printf("%d\n", soma);
    return 0;
}
